packages/discovery-provider/src/solana/solana_helpers.py

```diff
@@ -39,5 +39,8 @@ def get_derived_address(base, hashed_eth_pk, spl_token_id):
 # Static Memo Program ID
 MEMO_PROGRAM_ID = "Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo"
 
+# Static Memo Program ID
+MEMO_V2_PROGRAM_ID = "MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr"
```

Comment:
I could have sworn that somewhere we _were_ using memov2. But maybe that's just on the client side for Coinflow transactions.